update: Datadog integration withdrawal for startup-2 plans (#718)
aiven/aiven-docs: docs/integrations/datadog.md

[customise the metrics sent to Datadog](/docs/products/kafka/howto/datadog-customised-metrics).
:::

:::note
Datadog integration is not available for new Startup-2 plans in Aiven for Apache Kafka.

> Upgrade to a Business-4 plan or higher to use Datadog. Existing services with

Reviewer: Customers who have existing plans will be able to create new ones. 

Also add a sentence at the end: "It is recommend that all customers use business-4 plans and above for Kafka clusters with Datadog integration, due to the resource pressures this can place on the startup cluster

[customise the metrics sent to Datadog](/docs/products/kafka/howto/datadog-customised-metrics).
:::

:::note
Datadog integration is not available for new Startup-2 plans in Aiven for Apache Kafka.

> Existing customers using Startup-2 with Datadog integration can continue to create new
> Startup-2 plans with Datadog integration and use their existing services without
> upgrading to a higher plan.

Reviewer 2: Consider replacing "customers" with addressing them directly using "you":

```suggestion
If you have already used Startup-2 with Datadog integration, you can continue to create new
Startup-2 plans with Datadog integrations and use your existing services without
upgrading to a higher plan.
```

Author: I used 'Existing customers' instead of 'you' to maintain a more neutral and formal tone, as this applies to a specific group of users rather than addressing all readers directly. This also keeps consistency with other similar documentation.

Aiven recommends using a **Business-4 plan or higher** for Aiven for Apache Kafka
services with Datadog integration to avoid resource pressure on Startup-2 plans.

If you are an existing customer and cannot create a Startup-2 plan with Datadog
integration in a new project, contact [Aiven Support](mailto:support@aiven.io).
:::

> 

Reviewer 3: Could you turn this into a reusable component?
